hydra-editor.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency "almond-rails", '~> 0.1'
   s.add_dependency "cancancan", "~> 1.8"
   s.add_dependency "rails", ">= 5", "< 6"
-  s.add_dependency "simple_form", '~> 4.1.0'
+  s.add_dependency "simple_form", '>= 5.0'
```

Comment:
I think dropping support for `4.1.0` would require a major version. `s.add_dependency "simple_form", '>= 4.1.0', '< 6.0'` instead?